sinara.py

```diff
@@ -60,7 +60,8 @@ class Sinara(_SinaraTuple):
         "Clocker",
         "Booster",
         "Booster_Channel",
-        "DIO_MCX"
+        "DIO_MCX",
+        "Kasli_soc"
```

Comment:
```suggestion
        "Kasli_SoC"
```

Reply:
This comes from the .json like this. I used the exactly the same naming there as the m-labs template. Not sure if `Kasli_SoC` in the .json would work for building the firmware..